tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -418,32 +418,44 @@ function kbs_k8s_deploy() {
 # otherwise the cluster IP.
 #
 kbs_k8s_svc_host() {
+	local host
+
+	# Ingress
```

Comment:
For s390x, I got 

```
$ KBS_SVC_NAME="kbs"
$ KBS_NS="coco-tenant"
$ kubectl get svc "$KBS_SVC_NAME" -n "$KBS_NS" -o jsonpath='{.spec.clusterIP}'
10.43.140.148
```

Based on your logic, I've confirmed that adding:

```
host=""
```

here resolves the CI issue. Thanks.

Reply:
My bad, I've fixed the unbound variable like you mentioned here. Tests running.